docs/source/reference.adoc

```diff
@@ -1,3 +1,126 @@
 = Reference Documentation
+:imagesdir: ./assets
+:toc:
 
-This is the Reference documentation for fabric8-auth.
+This is the Reference documentation for fabric8-auth. You may find it useful if you wish to deploy and configure the service, or to interact
+with its authentication and authorization services via its REST endpoints.  If you wish to understand more about the inner workings of fabric8-auth and be
+able to build and/or develop the project, please refer to the link:developer.html[Developer documentation] instead.
+
+== Service Accounts
+
+Service accounts are used to authenticate other services in order for them to interact with the fabric8-auth permission (and other) endpoints.
```

Comment:
It's not for interacting with fabric8-auth endpoints only. It's for service-to-service communication in general.